modules/web/src/app/wizard/step/applications/component.ts

```diff
@@ -82,16 +82,18 @@ export class ApplicationsStepComponent extends StepBase implements OnInit, OnDes
         return application;
       }) || [];
 
-    merge(this._clusterSpecService.datacenterChanges, this._applicationService.applicationChanges)
-      .pipe(takeUntil(this._unsubscribe))
-      .subscribe(_ => {
-        this.loadDefaultAndEnforcedApplications();
-      });
+    this._clusterSpecService.datacenterChanges.pipe(takeUntil(this._unsubscribe)).subscribe(_ => {
+      this._applicationService.applications = [];
```

Comment:
- Enforced applications without a datacenter selector still show 

Example: Nginx was enforced and not tied to specific datacenter it is being skipped ATM

<img width="791" height="614" alt="Image" src="https://github.com/user-attachments/assets/9009a584-a54c-4c7d-931e-15f6912ce372" />

<img width="781" height="488" alt="Image" src="https://github.com/user-attachments/assets/5f9c7421-afcb-402d-be55-74689b7b0a22" />